thorlcr/activities/project/thprojectslave.cpp

```diff
@@ -255,10 +251,10 @@ class CPrefetchProjectSlaveActivity : public CSlaveActivity
         preload = helper->getLookahead();
```

Comment:
Theoretically this could also be dynamic (e.g., based on a stored variable) => move to start()